bokehjs/src/lib/models/widgets/file_input.ts

```diff
@@ -3,8 +3,11 @@ import {Widget, WidgetView} from "models/widgets/widget"
 
 export class FileInputView extends WidgetView {
   model: FileInput
-
+  
   protected dialogEl: HTMLInputElement
+  mime_type: string[] = []
+  filename: string[] = []
+  value: string[] = []
```

Comment:
Those are unused.

Reply:
True, those are artefacts from an earlier attempt.
I'll remove them.

Reply:
Done in PR #9732